### PR TITLE
Documentation: add qingcloud to production user

### PR DESCRIPTION
**Pull request by jolestar**
add qingcloud to production user

### CODE REVIEW

**Comment by xiang90** [comment]
LGTM.

/cc @heyitsanthony (can we bypass the commit message checking for this sort of thing? and merge it manually?)

**Comment by heyitsanthony** [comment]
@xiang90 there's no rush to merge these...

@jolestar could you please update the commit message to `Documentation: add qingcloud to production user` so it matches the commit format in https://github.com/coreos/etcd/blob/master/CONTRIBUTING.md#format-of-the-commit-message? Thanks!

File: Documentation/production-users.md [hunk 1]
```diff
@@ -69,3 +69,17 @@ Radius Intelligence uses Kubernetes running CoreOS to containerize and scale int
 
 [teamcity]: https://www.jetbrains.com/teamcity/
 [raoofm]:https://github.com/raoofm
+
+## QingCloud
+
+- *Application*: [QingCloud](qingcloud) appcenter cluster for service discovery as [metad](metad) backend.
+- *Launched*: December 2016
+- *Cluster Size*: 1 clusters of 3 members per user.
```
Comment (fanminshi) [inline Documentation/production-users.md]: `1 clusters` => `1 cluster `?

**Comment by xiang90** [comment]
@jolestar 

Hi, the commit message needs to be updated not the pull request title.

You can do 

```
git commit --amend

change message to [doucments: add qingcloud to production-users.md]

git push ... -f
```

**Comment by jolestar** [comment]
@xiang90  done

**Comment by xiang90** [comment]
LGTM. I squashed the two commits into one, and get it merged.

**Pull request merged**